src/main/java/Board.java

```diff
@@ -11,13 +11,13 @@ public Boolean isEmpty() {
         return !grid.contains("X") && !grid.contains("O");
     }
 
-    public Object markBoard(int position, String mark) {
-        return grid.set(position - 1, mark);
+    public void markBoard(int position, String mark) {
+       grid.set(position - 1, mark);
```

Comment:
How many places is ```position-1``` duplicated? Are there any instances of that duplication outside of this class?

Reply:
✅ now only one instance within a single class - PR #6